drivers/windows/R/batch.R

```diff
@@ -30,13 +59,14 @@ template_data_task_run <- function(task_id, config, path_root) {
   data$task_id_2 <- substr(task_id, 3, nchar(task_id))
 
   data$hipercow_library <- paste(
-    remote_path(file.path(path_root, config$path_lib), config$shares),
+    remote_path(file.path(path_root, config$path_lib), config$shares,
+                platform == "linux"),
```

Comment:
why are we passing a boolean here for linux rather than just `platform`? The latter seems more general and means we don't have to think about if we're dealing with it booleanised or not

Reply:
Done. I might go a bit further, since having three occurrences of config$ on this line looks a bit overkill